extensions/amp-analytics/amp-analytics.md

```diff
@@ -729,6 +729,10 @@ interval. Use `timerSpec` to control when this will fire.
   will be infinity.
 - `immediate` trigger timer immediately or not. Boolean, defaults to true
 
+[tip type="note"]
+If `maxTimerLength` has not been defined, the trigger will continue to send out pings until the page is fully closed (and not just hidden).
```

Comment:
We have  "Unless a `startSpec` is specified, the timer will trigger immediately (by default, can be unset) and then at a specified interval thereafter." above, do you think it's a good idea to add the note there as well. 

Also I think it would be helpful to add an example on how to start/stop timer on document visible/hidden. 

Something like
"The timer will trigger continuously regardless of the document state (hidden or inactive), until `maxTimerLength` has been reached or `stopSpec` specification has been met.........
Refer to the example......."

Reply:
> do you think it's a good idea to add the note there as well.

Do you mean move the note under that paragraph or create a duplicate note there as well?

> Also I think it would be helpful to add an example on how to start/stop timer on document visible/hidden.

Agreed.